packages/odyssey-storybook/src/components/Field/Field.stories.tsx

```diff
@@ -20,7 +20,6 @@ export default {
   title: `Components/Field`,
   component: Source,
   parameters: {
-    layout: "fullscreen",
```

Comment:
I cant remember what this was for, do you?

Reply:
We had a mix of layouts previously. I've tried to update them based on the actual component needs/display in this PR. If it's not a fullscreen component like Banner, I've removed this param.